app/src/main/java/com/genymobile/pr/ui/MainActivity.java

```diff
@@ -20,18 +17,30 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Fragment fragment;
         SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
-        if (preferences.contains(getString(R.string.pref_login)) &&
-                preferences.contains(getString(R.string.pref_password))) {
-            fragment = new PullRequestListFragment();
+
+        boolean credentialsSet = preferences.contains(getString(R.string.pref_login)) &&
+                preferences.contains(getString(R.string.pref_password));
+        boolean organizationSet = preferences.contains(getString(R.string.pref_organization));
+
+        if (credentialsSet) {
+            if (organizationSet) {
+                fragment = new PullRequestListFragment();
```

Comment:
[minor] Just thinking, in a future PR, we should rename this to have PullRequest**s**ListFragment (plural)